main.cpp

```diff
@@ -5,7 +5,7 @@
 
 
 namespace {
-
+//..........................Generic UI Base Class.............................//
```

Comment:
Usually we avoid these type of comments, except when the file becomes really complicated. In fact, this is mostly paraphrasing the class name, so it is not adding any value.

Reply:
ok. I am going to fix it